AttitudeManager/FlightModes/Inc/fbwa.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef ZPSW3_AM_FBWA_HPP
+#define ZPSW3_AM_FBWA_HPP
+
+#include "flightmode.hpp"
+
+namespace AM {
+
+class FBWA : public Flightmode {
+   public:
+    FBWA() = default;
+
+    AttitudeManagerInput run(const AttitudeManagerInput& input) override;
+    void updatePid() override;
+    void updatePidGains();
+};
+
+}  // namespace AM
+
+#endif  // ZPSW3_AM_FBWA_HPP
```

Comment:
```suggestion
#endif  // ZPSW3_AM_FBWA_HPP

```

remember to have newlines at EOF